<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaserOptionWidget</class>
 <widget class="QWidget" name="LaserOptionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>904</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="12" column="1">
      <widget class="QLabel" name="_autoAngleStartLbl">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Angle depart :</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLineEdit" name="_autoAngleStartEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLabel" name="_autoAngleEndLbl">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Angle arriv√© :</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QSlider" name="_powerEdit">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="maximum">
        <number>2000</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Rouge :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QSlider" name="_angleEdit">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="maximum">
        <number>36000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QCheckBox" name="_automaticRotationCheckbox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Rotation automatique</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLabel" name="_autoAngleStepLbl">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Step :</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLineEdit" name="_autoAngleStepEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLineEdit" name="_autoAngleEndEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QSlider" name="_colorBlueEdit">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Vert :</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="_chooseColorButton">
       <property name="text">
        <string>Choisir couleur</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="_angleLabel">
       <property name="text">
        <string>Angle : </string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QSlider" name="_colorGreenEdit">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Bleu :</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QSlider" name="_colorRedEdit">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Power : </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="_objectName"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
